onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/BookingRule.java

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright (C) 2020 Victor Pavlushkov <victor@kyyti.com>
```

Comment:
```suggestion
 * Copyright (C) 2020 Kyyti Group Ltd
```
Similar change should be applied to some of @hannesj 's new files too in gtfs-flex-2 branch.